app.rb

```diff
@@ -0,0 +1,92 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    puts 'Welcome to School Library App!'
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each do |object|
+      puts "Title: '#{object.title}', Author: #{object.author}"
+    end
+  end
+
+  def list_people
+    @people.each do |object|
+      puts "[#{object.class}] Name: #{object.name}, ID: #{object.id}, Age: #{object.age}"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    option_person = gets.chomp.to_i
+    case option_person
+    when 1
+      print 'Age: '
+      person_age = gets.chomp.to_i
+      print 'Name: '
+      person_name = gets.chomp.to_s
+      print 'Has parent permission? [Y/N]: '
+      person_permission = gets.chomp.downcase
+      if person_permission.capitalize == 'N'
+        @people.push(Student.new(age: person_age, name: person_name, parent_permission: false))
+      else
+        @people.push(Student.new(age: person_age, name: person_name))
+      end
```

Comment:
- [ ] @enmanuelbayen  Great work up to this point. However, when trying to create a student, I encountered a small error which is fixable.
![image](https://github.com/enmanuelbayen/OOP-school-library/assets/98453979/af3f9d60-c778-48c7-9a85-081ca4dd4630)

- [ ] When instantiating your student class in `app class`, you used `keyword format ` to pass arguments .

```ruby
  Student.new(age: person_age, name: person_name, parent_permission: false)
```
- [ ] For this to work, you need to update your `student ` class initialize method to use keyword arguments for all the parameters as well
  ### Here is an example

```ruby
class Student
  def initialize(id: rand(1..100), name: 'Unknown', age: nil, parent_permission: true)
    # Your initialization code here
  end
end
```
- Keep up the good work 👍